performance/tests/sandstone/ListItemsTests.js

```diff
@@ -34,12 +36,38 @@ const listItemTests = (componentName, dataSize) => describe(componentName, () =>
 		});
 	});
 
+	describe('ScrollButton', () => {
+		it('scrolls down with translate scrollMode', async () => {
```

Comment:
'describe' is duplicated. Please move this line to ScrollButton 'describe' above. Agate is doing well.